### ISSUE TITLE
controller: error when logging in

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#368]
* r5_fixes
f3f13d1

When logging on the controller, i get the following error:

```
[root@master ~]# ssh -Y node002
Last login: Tue Jul 28 16:45:44 2015 from 192.168.1.254
-bash: /root/.bashrc: line 14: unexpected EOF while looking for matching `"'
-bash: /root/.bashrc: line 15: syntax error: unexpected end of file
```

**Comment by hansthen** [comment]
Fixed in r5_fixes. Manual workaround add the missing quotes to the end of the string as per the bug report.

**Comment by themisa** [comment]
Fixed in **Version:** r5_fixes e5f4623
